packages/middleware-multipart/src/specs/multipart.util.spec.ts

```diff
@@ -0,0 +1,134 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { createHttpRequest } from '@marblejs/core/dist/+internal/testing';
+import { shouldParseFieldname, shouldParseMultipart, setRequestData, streamFileTo } from '../multipart.util';
+import { FileIncomingData } from '../multipart.interface';
+import { from } from 'rxjs';
+
+test('#shouldParseFieldname checks if given fieldname can be parsed by middleware', () => {
+  expect(shouldParseFieldname(undefined)('test')).toEqual(true);
+  expect(shouldParseFieldname(['test_1'])('test_1')).toEqual(true);
+  expect(shouldParseFieldname(['test_1'])('test_2')).toEqual(false);
+  expect(shouldParseFieldname([])('test')).toEqual(false);
+});
+
+test('#shouldParseMultipart checks if incoming request can be parsed by middleware', () => {
+  // given
+  const multipartRequest = createHttpRequest({
+    method: 'POST',
+    headers: { 'Content-Type': 'multipart/form-data; boundary=----WebKitFormBoundary7MA4YWxkTrZu0gW'},
```

Comment:
How about `'Content-Type': 'multipart/form-data'`, but with boundary missing. What happens when taht happens?

Reply:
I think that this case is handled internally in `busboy`